draft-ietf-taps-impl.md

```diff
@@ -533,7 +533,7 @@ The effect of the application sending a Message is determined by the top-level p
 
 - Ordered: when this is false, this disables the requirement of in-order-delivery for protocols that support configurable ordering. When the protocol stack does not support configurable ordering, this property may be ignored.
 
-- Safely Replayable: when this is true, this means that the Message can be used by mechanisms that might transfer it multiple times -- e.g., as a result of racing multiple transports or as part of TCP Fast Open. Also, protocols that do not protect against duplicated messages, such as UDP, can only be used with Messages that are Safely Replayable.
+- Safely Replayable: when this is true, this means that the Message can be used by a transport mechanism that might transfer it multiple times -- e.g., as a result of racing multiple transports or as part of TCP Fast Open. Also, protocols that do not protect against duplicated messages, such as directlty layered over UDP, can only be used with Messages that are Safely Replayable. When a transport system is permitted to replay messages, replay protection could be provided by the application.
```

Comment:
```suggestion
- Safely Replayable: when this is true, this means that the Message can be used by a transport mechanism that might transfer it multiple times -- e.g., as a result of racing multiple transports or as part of TCP Fast Open. Also, protocols that do not protect against duplicated messages, such as UDP (when used directly, without a protocol layered atop), can only be used with Messages that are Safely Replayable. When a transport system is permitted to replay messages, replay protection could be provided by the application.
```

Just a suggestion, fine by me to phrase this differently - but now, you have:
- typo directlty
- "protocols that do not protect against duplicated messages, such as directly layered over UDP" ... is this really okay to write? It reads awkwardly to me ... I mean, such as asks for, what, a protocol? So this would mean "such as protocols that are directly layered over UDP?" but QUIC is such a protocol. This is about the application being layered directly over UDP!  So ...   hence my suggestion above.